apisix/cli/ops.lua

```diff
@@ -503,38 +501,27 @@ Please modify "admin_key" in conf/config.yaml .
     yaml_conf.apisix.ssl.listen = ssl_listen
     yaml_conf.apisix.enable_http3_in_server_context = enable_http3_in_server_context
 
-
-    if yaml_conf.apisix.ssl.ssl_trusted_certificate ~= nil then
-        local cert_paths = {}
-        local ssl_certificates = yaml_conf.apisix.ssl.ssl_trusted_certificate
-        for cert_path in string.gmatch(ssl_certificates, '([^,]+)') do
-            cert_path = util.trim(cert_path)
-            if cert_path == "system" then
-                local trusted_certs_path, err = util.get_system_trusted_certs_filepath()
-                if not trusted_certs_path then
-                    util.die(err)
-                end
-                table.insert(cert_paths, trusted_certs_path)
-            else
-                -- During validation, the path is relative to PWD
-                -- When Nginx starts, the path is relative to conf
-                -- Therefore we need to check the absolute version instead
-                cert_path = pl_path.abspath(cert_path)
-                if not pl_path.exists(cert_path) then
-                    util.die("certificate path", cert_path, "doesn't exist\n")
-                end
-
-                table.insert(cert_paths, cert_path)
-            end
+    if not yaml_conf.apisix.ssl.ssl_trusted_certificate then
+        yaml_conf.apisix.ssl.ssl_trusted_certificate = "system"
```

Comment:
As it relates to default value changes, this should be added to the default values table in `apisix/cli/config.lua` rather than only setting the variable directly here. if that table is also used here, we can reduce codes.

Reply:
- Add default to `apisix/cli/config.lua`
- Check if `ops.lua` is using `conf/config.yaml` and `apisix/cli/config.lua` aggregation
  -  if so, remove the assignment here

Reply:
done